From 6fbc109ee74dc46bd2b29162bfd7d9a28448189d Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Thu, 29 Aug 2024 10:42:23 +0530
Subject: [PATCH] net: disable Werror temporarily

This change disables Werror for net & drivers/net directory
temporarily.

Change-Id: Ia5a68d1afd766e7950f7447cb6416514ac3cd475
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 drivers/net/Makefile | 1 +
 net/Makefile         | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/Makefile b/drivers/net/Makefile
index 6ce076462dbf..caf0a6b7f97f 100644
--- a/drivers/net/Makefile
+++ b/drivers/net/Makefile
@@ -6,6 +6,7 @@
 #
 # Networking Core Drivers
 #
+KBUILD_CFLAGS := $(filter-out -Werror, $(KBUILD_CFLAGS))
 obj-$(CONFIG_BONDING) += bonding/
 obj-$(CONFIG_IPVLAN) += ipvlan/
 obj-$(CONFIG_IPVTAP) += ipvlan/
diff --git a/net/Makefile b/net/Makefile
index 57ddb1744ea3..3fcb488fc578 100644
--- a/net/Makefile
+++ b/net/Makefile
@@ -5,7 +5,7 @@
 # 2 Sep 2000, Christoph Hellwig <hch@infradead.org>
 # Rewritten to use lists instead of if-statements.
 #
-
+KBUILD_CFLAGS := $(filter-out -Werror, $(KBUILD_CFLAGS))
 obj-y				:= devres.o socket.o core/
 
 obj-$(CONFIG_COMPAT)		+= compat.o
-- 
2.34.1

